sources/packages/web/src/assets/css/vuetify.scss

```diff
@@ -1,4 +1,4 @@
-@import "./global-style-variables.scss";
+@import "./base.scss";
```

Comment:
Why are we importing the base.scss into vuetify.scss? is it to use and extend some base css classes?

Reply:
class  `.label-bold` and `.nav-item-label` are imported from `base.scss`.  `global-style-variables.scss` is already impoerted in `base.scss`.  ![image](https://user-images.githubusercontent.com/77353155/186252393-f7fffb50-43ed-499f-9ef1-d1203117ef18.png)